### PR TITLE
Interactive transcripts display

### PR DESCRIPTION
**Pull request by dorosh**
get transcripts for youtube, wistia, brightcove
- add transcripts to context template players
- add button for subtitles and text block with translates

### CODE REVIEW

File: video_xblock/backends/youtube.py [hunk 1]
```diff
@@ -31,15 +31,16 @@ def get_frag(self, **context):
...
+                # "videoJsResolutionSwitcher": {
+                #     "dynamicLabel": True,
+                #     "default": "low"
+                # },
                 "xblockEventPlugin": {}
             }
         })
 
         frag = super(YoutubePlayer, self).get_frag(**context)
+        print(context)
```
Comment (z4y4ts) [inline video_xblock/backends/youtube.py]: Either `log.debug()` or remove

File: video_xblock/backends/youtube.py [hunk 2]
```diff
@@ -31,15 +31,16 @@ def get_frag(self, **context):
             "youtube": {"iv_load_policy": 1},
             "playbackRates": [0.5, 1, 1.5, 2],
             "plugins": {
-                "videoJsResolutionSwitcher": {
-                    "dynamicLabel": True,
-                    "default": "low"
-                },
+                # "videoJsResolutionSwitcher": {
```
Comment (z4y4ts) [inline video_xblock/backends/youtube.py]: Delete completely, add a note when to revert back or don't touch it.

File: video_xblock/static/html/youtube.html [hunk 3]
```diff
@@ -5,4 +5,9 @@
   width="829" height="511"
   data-setup='{{ data_setup }}'
 >
+  {% for caption in player_state.transcripts %}
```
Comment (z4y4ts) [inline video_xblock/static/html/youtube.html]: Why single element of `transcripts` is `caption`?

Here and in similar places

**Comment by z4y4ts** [comment]
Also, please update `bower.json`

File: video_xblock/static/html/youtube.html (hunk 3 above)
Comment (dorosh) [inline video_xblock/static/html/youtube.html]: i will rename to var caption  to transcript

**Pull request merged**